box-ui/src/components/ts/JitsiComponent.tsx

```diff
@@ -68,6 +79,19 @@ const instantiateJitsi = async (
 const JitsiMeetComponent: FunctionComponent<InputRoomProps> = (props: InputRoomProps) => {
     const [displayHangup, setDisplayHangup] = useState<boolean>();
     const navigate = useNavigate();
+    const [raised, setRaised] = useState(false);
+    const [idTo, setidTo] = useState();
+    function lowerHand() {
+        setRaised((raised) => (raised ? !raised : raised));
+    }
+    function switchState() {
+        console.log(raised);
+        setidTo((idTo) => idTo);
+        setRaised((raised) => !raised);
+    }
+    useEffect(() => {
+        clearTimeout(idTo);
+    }, [raised]);
```

Comment:
Not sure why you need this.

Reply:
Everytime a hadn't is raised, the current timer that calls the lowerhand() function should be cleared so not as not to interfere with the new timer. I should add a if !raised condition though